includes/expressAlarm.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+expressAlarm is the function to enable BOSwatch to deal with Swissfone Express-Alarm
```

Comment:
Gibt es zum `Swissfone Express-Alarm` irgend ein offizielles Dokument, welches man als Referenz evtl direkt im Kommentar verlinken könnte?

Reply:
Mein wissen darüber ist aus vielen Beträgen in den einschlägigen Foren zusammengesucht. Ich kann mal schauen ob ich da noch etwas zusammenhängendes finde. Swissphone hält die Infos dazu leider ziemlich unter Verschluss.

Reply:
Eine prinzipille Beschreibung wie EA abläuft wäre aber auf jeden Fall nicht verkehrt, damit sich jemand der es nicht kennt was drunter vorstellen kann ;-)
Kann mir auch vorstellen, dass es viele in Ihren Funkkreisen haben aber nichts davon, bzw dessen Funktion wissen.

Reply:
Ich habe mir mal eine Beschreibung einfallen lassen. Außerdem habe ich mir überlegt, dass es vermutlich besser wäre, wenn ich den von Swissphone geschützten Markennamen Express-Alarm aus dem Code entferne. Da sich damit sehr viel geändert hat, bin ich aber unsicher, ob ich den neuen Code einfach hinzufügen, oder ob ich einen neuen branch und pull request machen soll.
Was ist denn das sinnvollste vorgehen?

Reply:
hinzufügen... sollte es ein problem geben, kann man ja ohne Aufwand zum jetzigen Stand reverten ;-)